lnwallet/channel.go

```diff
@@ -1869,7 +1870,7 @@ func (lc *LightningChannel) restoreCommitState(
 	lc.localCommitChain.addCommitment(localCommit)
 
 	lc.log.Tracef("starting local commitment: %v",
-		newLogClosure(func() string {
+		lnutils.NewLogClosure(func() string {
```

Comment:
I'm thinking it could be good to add to the `lnutils` package a function specifically designed to handle the spew case

```golang
func SpewLogClosure(a interface{}) func() string {
        return func() string {
                return spew.Sdump(a)
        }
}
```

Reply:
cool I like it